### DISCUSSION TITLE
help for SignatureAlgorithms format in Security Configuration

### DISCUSSION THREAD

**Discussion by kaoriakana** [onvif/testspecs#68] [Q&A] [answered]
Hello !
I get a step failed in Security Configuration\Capabilities\ADVANCED_SECURITY-5-1-2-v15.06 Get Services and Get Security Configuration Service Capabilities Consistency.

The pictures are about step details and the key contents.
![steps](https://github.com/onvif/testspecs/assets/42507907/97f49a1f-045d-4e1b-9a0d-255cf0d17526)

![getservices](https://github.com/onvif/testspecs/assets/42507907/087e7ab9-caf7-4b82-a084-6787a6557413)

![getcapabilities](https://github.com/onvif/testspecs/assets/42507907/26db415d-5451-463c-97ea-68e8f40c4162)

Could you kindly give me some information to modify the SignatureAlgorithms.

Thank you.

Akana

**Comment by madhurao68** [comment, marked as answer]
Hi,
Both GetServiceCapabilities (from namespace http://www.onvif.org/ver10/advancedsecurity/wsdl) and GetServices have to use the same type Capabilities (from namespace http://www.onvif.org/ver10/advancedsecurity/wsdl). So they must have the same structure.

In this case, GetServiceCapabilities has a valid structure, but the structure for GetServices is incorrect. The attribute SignatureAlgorithms.algorithm is unknown by the schema and hence not recognized as SignatureAlgorithms.

**Comment by kaoriakana** [comment]
Hi! Madhu:
I fix this error from the cue  you give.

Thank you so much.
